cli/src/main.rs

```diff
@@ -0,0 +1,8 @@
+#[tokio::main]
+async fn main() -> color_eyre::Result<()> {
+    let _db_pool = db::setup_db_pool().await?;
```

Comment:
You'll be able to use this `db_pool` with `sqlx` to access the Postgres database! Not used yet in the CLI but should be setup and ready for ya